### ISSUE TITLE
Application Information is web application specific

### ISSUE THREAD

**Issue by omz13** [indieweb/indieauth#123]
In [4.2.1](https://indieauth.spec.indieweb.org/#application-information) it is implicit that the `application` is a web application.

This should be re-written to take into account that there are device-based applications too.

**Comment by sknebel** [comment]
No, it merely implies that a website about the application exists?

**Comment by omz13** [comment]
Yes, it implies/is implicit. I'm simply saying it should be made explicit (since the world does not only comprise web-based apps). And, if you have a device app it means you have to have a corresponding web site so the AS can have a source of `mf2+html` to work with when presenting the user their options per [§4.2.1 IndieAuth](https://indieauth.spec.indieweb.org/#client-information-discovery).

This was originally raised by somebody on SocialCG who bothered to read IndieAuth with a fresh set of eyes.

**Comment by omz13** [comment]
@jalcine The logic goes: all clients are web apps, therefore there is always a website from which an AS can find some mf2+html and parse out some app info (h-app) to present to the user during the authorization flow. As we both have device apps, this is where this logic breaks down. As I have my own AS and device-based apps I can and do extend and deviate from the spec because that way I get to have a working solution.

**Comment by dshanske** [comment]
@omz13 I concur with @sknebel . It doesn't assume there is a webapp. It assumes there is a website about an application that can be used as its client_id and that URL is where application specific data.

It never says that the client must be a web app, just that the ID must be a URL and application information should be discoverable at that URL.  

Is your request that it expressly say that the URL is solely for this purpose and the application does not need to function from that location?
